packages/docs/docs/docs/upgrading.md

```diff
@@ -160,6 +160,58 @@ truffle(local)> myContract.x().then(x => x.toString())
 43
 ```
 
+Now let's imagine that instead of just adding a new 
+function to the contract (a change to functionality), we wanted to add a new 
+variable `t` to our contract. But how do we set the initial value of `t`?
+The variables `x` and `s` were initialized with the `initialize` function,
+which was called when the proxy was created via the `zos create MyContract --init initialize --args ...` 
+command. At first, you may think that the solution would be to add the initialization of `t`
+to the end of the initialize function, but that wouldn't work. We cannot use the same `initialize` function, because
+the `Initializable` modifier guards it against being called more than once. We need a new function. 
+
+The `update` command also accepts `--init` and `--args` parameters, so we can use a function
+with it to initialize the new variable. A good name for the 
+new function could be something like `initializeT`. This function would simply
+set the initial value of `t` and check that it has not yet been initialized. It should be called with `zos update MyContract --init initializeT --args 99`.
+
+```
+function initializeT(uint256 _t) public {
+  require(_t == 0);
+  t = _t;
+}
+```
+
+This initialization validation for `t`, of course, would only make sense if `t` cannot be zero. 
+
+The resulting code would be:
+
+```solidity
+pragma solidity ^0.5.0;
+
+import "zos-lib/contracts/Initializable.sol";
+
+contract MyContract is Initializable {
+
+  uint256 public x;
+  string public s;
+  uint256 public t;
+
+  function initialize(uint256 _x, string memory _s) initializer public {
```

Comment:
The original initialize should also be updated to include `t`, so any new instances can be fully initialized; otherwise any new deploy would require calling initialize + initializeT. 

I'd suggest also to make a note about this: perhaps earlier when you say "At first, you may think that the solution would be to add the initialization of `t` to the end of the initialize function, but that wouldn't work", we could clarify that it won't work for updating, but it's still needed for new instances.